app/views/pages/device_status.html.erb

```diff
@@ -0,0 +1,224 @@
+<ul class="nav nav-tabs mb-4">
+  <li class="nav-item">
+    <a class="nav-link active" href="#">Device Status</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link disabled" href="#">Analaytics and Recommendations</a>
+  </li>
+</ul>
+
+<div class="row">
+	<div class="col-md-12 mb-4">
+		<div class="card box-shadowed">
+			<div class="card-body">
+				<div class="system-wrapper">
+					<div>
+						<%= image_tag('regions/Image 1.png', alt: "Edit Entry") %>
+					</div>
+					<div>
+						<div class="system-box p-3">
+				    		<div class="text-center">
+				    			<div class="system-box-title">System Name</div>
+				    			<p class="font-weight-bold">AHU1</p>
+				    		</div>
+				    		<div class="text-center">
+				    			<div class="system-box-title">Status</div>
+				    			<p class="font-weight-bold status-on">On</p>
+				    		</div>
+				    	</div>
+				    	<div class="system-box p-3">
+				    		<div class="text-center">
+				    			<div class="system-box-title">Supply Temperature</div>
+				    			<p class="font-weight-bold">27 C</p>
+				    		</div>
+				    		<div class="text-center">
+				    			<div class="system-box-title">Return Temperature</div>
+				    			<p class="font-weight-bold">28 C</p>
+				    		</div>
+				    	</div>	
+					</div>
+					<div>
+						<div class="system-box p-3">
+				    		<div class="text-center">
+				    			<div class="system-box-title">Setpoint Temperature</div>
+				    			<p class="font-weight-bold">27 C</p>
+				    		</div>
+				    		<div class="text-center">
+				    			<div class="system-box-title">Mode</div>
+				    			<p class="font-weight-bold">Cool</p>
+				    		</div>
+				    	</div>
+				    	<div class="system-box p-3">
+				    		<div class="text-center">
+				    			<div class="system-box-title">Ambient Temperature</div>
+				    			<p class="font-weight-bold">27 C</p>
+				    		</div>
+				    		<div class="text-center">
+				    			<div class="system-box-title">Ambient Humidity</div>
+				    			<p class="font-weight-bold">28%</p>
+				    		</div>
+				    	</div>	
+					</div>
+					<div class="system-box-four text-center">
+						<div class="system-box-four-title p-3">
+				    		<div class="system-box-title">Fan Speed</div>
+				    		<p class="font-weight-bold">2</p>
+				    	</div>
+					</div>
+					<div class="system-box-five icon-three-dots"></div>
+				</div>
+
+				
+			</div>
+			<div class="card-footer position-relative bg-purple text-muted">
```

Comment:
like i said above, use this for the footer
```html
<div class="card-footer position-relative bg-purple">
<div class="row align-items-center">
    <div class="col text-muted">
    <span >Zone A</span><span class="ml-5">ODU# 1</span>
    </div>
    <div class="col text-right">
      <button type="button" class="btn btn-outline-purple px-3">Adjust</button>
    </div>
  </div>
  </div>
```

Reply:
ok let me see if it works.